src/main/java/jenkinsci/plugins/influxdb/generators/SonarQubePointGenerator.java

```diff
@@ -49,29 +48,34 @@
     private static final short DEFAULT_MAX_RETRY_COUNT = 10;
     private static final int DEFAULT_RETRY_SLEEP = 5000;
 
-    private static final String URL_PATTERN_IN_LOGS = ".*" + Pattern.quote("ANALYSIS SUCCESSFUL, you can browse ")
-            + "(.*)";
-    private static final String TASK_ID_PATTERN_IN_LOGS = ".*" + Pattern.quote("More about the report processing at ")
-            + "(.*)";
+    // patterns used for data extraction from ${WORKSPACE}/${SONAR_BUILD_REPORT} file
+    private static final String SONAR_BUILD_REPORT = "/build/sonar/report-task.txt";
```

Comment:
This looks dangerous.

SonarQube's [official documentation](https://docs.sonarqube.org/latest/analysis/analysis-parameters/) says the default value for `sonar.scanner.metadataFilePath` is the value for `sonar.working.directory`, which in turn has a default value of `.scannerwork`. I think the default value should correspond to that, but I think it could be overwritable with an environment variable.